packages/client/src/rpc/modules/eth.ts

```diff
@@ -1426,7 +1426,8 @@ export class Eth {
 
     if (this._chain.blockchain.common.isActivatedEIP(4844)) {
       baseFeePerBlobGas.push(
-        requestedBlocks[requestedBlocks.length - 1].header.calcNextBlobGasPrice(),
+        // use the last blocks common for fee estimation
+        requestedBlocks[requestedBlocks.length - 1].header.calcNextBlobGasPrice(requestedBlocks[requestedBlocks.length - 1].header.common),
```

Comment:
If there are two ore more blocks, we can use the common of `requestedBlocks.length -1` and we can call it on `requestedBlocks.length - 2` (this would only be observable at the fork block)

Reply:
used getBlobGasPrice instead of calcnext, see if that makes sense